core/modules/rproc/InfileMerger.cc

```diff
@@ -77,14 +77,19 @@ using lsst::qserv::proto::ProtoImporter;
 using lsst::qserv::proto::WorkerResponse;
 using lsst::qserv::rproc::InfileMergerConfig;
 using lsst::qserv::rproc::InfileMergerError;
+using lsst::qserv::util::ErrorCode;
```

Comment:
There is one use of `ErrorCode` and eight `util::ErrorCode` in this file, may be better stick to `util::ErrorCode` everywhere?